src/gui/tray/syncstatussummary.cpp

```diff
@@ -196,6 +231,33 @@ void SyncStatusSummary::onFolderSyncStateChanged(const Folder *folder)
     setSyncStateForFolder(folder);
 }
 
+void SyncStatusSummary::onFileProviderDomainSyncStateChanged(const AccountPtr &account, SyncResult::Status state)
```

Comment:
this needs to be wrapped inside an `#ifdef BUILD_FILE_PROVIDER_MODULE` too

Reply:
Oops, thanks!